source/Stargate-API-Skeleton/StargateApplication.class.st

```diff
@@ -186,6 +229,21 @@ StargateApplication >> createAPI [
 	^ api
 ]
 
+{ #category : #'private - activation/deactivation' }
+StargateApplication >> dumpDataOf: aZincEvent on: aLogRecordJson [
+
+	| eventJson |
+	eventJson := NeoJSONObject fromString:
+		             (String streamContents: [ :stream | 
+			              (NeoJSONWriter on: stream)
+				              mapZincEvent;
+				              nextPut: aZincEvent ]).
+	aLogRecordJson
+		at: #summary put: eventJson summary;
+		at: #request put: eventJson request;
+		at: #response put: eventJson response
+]
```

Comment:
We have to consider the possibility of something that can be mapped here, and producing an error log record in this case, but not killing the app because some log records cannot be produced.